src/templates/dashboard/types.ts

```diff
@@ -1,11 +1,11 @@
 import { DataTableProps } from '../../molecules/tables/types';
-import { CardListPorps } from '../../organisms/card-list';
+import { CardListProps } from '../../organisms/card-list';
 
 type DashboardExtraProps = {
-  withCards: boolean;
-  withTable: boolean;
+  showCards: boolean;
+  showTable: boolean;
```

Comment:
it would be nice to keep the same names without changing them since we have used this component in multiple places